container.inc

```diff
@@ -631,18 +631,47 @@ stock GetItemContainerSlot(Item:itemid, &slot) {
 
 ==============================================================================*/
 
-
-hook OnItemDestroy(Item:itemid) {
```

Comment:
What if this were simply changed to OnItemDestroyed, so it's only removed after the item has been destroyed rather than before?

Reply:
The behavior of scripts would change since at that point `IsValidItem` returns false, meaning `RemoveItemFromContainer` wouldn't return with `0` for the exit code.
Also the `OnItemRemoveFromContainer` and `OnItemRemovedFromContainer` events wouldn't be getting called anymore.